.github/ISSUE_TEMPLATE/code_improvement.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Code Improvement
+about: Create a request for improving the code quality of RMC
+labels: Cat: cleanup, Cat: enhancement 
+---
+<!--
+Please report security concerns to us on the
+[AWS vulnerability reporting](http://aws.amazon.com/security/vulnerability-reporting/) page.
+-->
+
+<!--
+Thank you for requesting an RMC improvment!
+Please include the following information if it applies to your case:
+-->
+**Requested improvement:**
+
+**Use case:**
+
+**Is this a breaking change?**
```

Comment:
Do this kind of issues
 1. require a use case?
 2. cause breaking changes?

Reply:
The use case I was thinking as existing use cases that trigger this code not new ones. I can remove it or rename it. Suggestions?

I think there may be cases where a code improvement can cause breaking changes. For example, deprecating some ugly options and stuff.

Reply:
I see, thanks. Maybe we could use "Related features" instead of "Use case", but I would expect "Requested improvement" to already include that information in most cases. But we can try with this and see how it goes.